src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2267,6 +2270,93 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly referenceGenes = remoteData<ReferenceGenomeGene[]>({
+        await: ()=>[
+            this.studies,
+            this._mutationEnrichmentDataWithoutCytoband,
+            this._copyNumberHomdelEnrichmentData,
+            this._copyNumberAmpEnrichmentData,
```

Comment:
this would be fetching both mutation and cna enrichments data together. this might cause delay in loading the page that is viewed. @adamabeshouse any suggestions?

Reply:
good point. one way to solve it is fetch and deal with mutation enrichment reference genes separately from cna enrichment reference genes.